### PR TITLE
feat: Small Fixes + Replay Tutorial button

### LINKED ISSUE #184: 🐾 Smol Update: Remove user controls on very last page of activity 1
request as per karen!
see #187 

* [ ] :wave: I may be able to implement this update request

---

This is a :feet: smol update

### LINKED ISSUE #187: 🐾 Smol Update: remove user controls on last page of activity 2
as per karen
see #184 

* [ ] :wave: I may be able to implement this update request

---

This is a :feet: smol update

### LINKED ISSUE #188: 🚀 Feature: Replay tutorials button
image
### Proposed Solution
- put      ` child: <GameIntroSlide2/>`, into the `Game/index.tsx`
- reset `statesToSkip` here https://github.com/uclaacm/Playnet/blob/217f922ba86f31a941bd467c3d1bc0048387fc97/src/components/Activities/Activity3/Game/index.tsx#L49

* [ ] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by darren2hang**
- Remove the sound buttons from the last page of Activity 1 and Activity 2
- Move the intro slide into the activity 3 Game component
- change game to start at intro slide and not officially start until the slide after the first slide

closes #184 
closes #187 
closes #188

### CODE REVIEW

File: .vs/ProjectSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (reginawang99) [inline .vs/ProjectSettings.json:1]: can we remove this folder / add it to the `.gitignore`

**Review by reginawang99 — CHANGES_REQUESTED** [review]
this also really isnt in scope, but would you mind removing controls on the last page of intro also?
image

File: src/components/Activities/Activity3/Game/GameIntroSlides.tsx [hunk 2]
```diff
@@ -2,21 +2,45 @@ import React from 'react';
...
+          <button className='playnet-button playnet-btn-blue'
+            onClick={() => {
+              storage.removeItem(SESSION_SKIP_STATES);
+              props.startNewGame();
+            }}
+          >
+            Replay Tutorial
+          </button>
+        }
+      </div>
```
Comment (reginawang99) [inline src/components/Activities/Activity3/Game/GameIntroSlides.tsx:43]: can we make this only in the right half? i think it looks a lil better that way! your choice tho after poking around!

**Comment by darren2hang** [comment]
> this also really isnt in scope, but would you mind removing controls on the last page of intro also?
> image

sure!

File: .vs/ProjectSettings.json (hunk 1 above)
Comment (darren2hang) [reply .vs/ProjectSettings.json]: yea, my bad. I pushed it by accident

File: src/components/Activities/Activity3/Game/GameIntroSlides.tsx (hunk 2 above)
Comment (darren2hang) [reply src/components/Activities/Activity3/Game/GameIntroSlides.tsx:43]: how would we do it if both the start button and the replay button are being displayed? Would both buttons be on the right? Or is it play game button on the right side and when both buttons are there we just have a button on each side?

Comment (reginawang99) [reply src/components/Activities/Activity3/Game/GameIntroSlides.tsx:43]: maybe both on the right? 
image

**Review by reginawang99 — APPROVED** [review]
LGTM!! 🎉

**Pull request merged**